### PR TITLE
Entitycreation

### PR DESCRIPTION
**Pull request by yllen**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | /no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #2598

New lang string

### CODE REVIEW

File: config/.htaccess [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<IfModule mod_authz_core.c>
```
Comment (trasher) [inline config/.htaccess]: Typo

Comment (yllen) [inline config/.htaccess]: ?? j'ai remis ce qui était dans 9.1/bugfixes

Comment (trasher) [inline config/.htaccess]: No, you write `<<` instead of `<`. There would be no diff here if files were the same :)

Comment (trasher) [inline config/.htaccess]: I've fixed.

**Pull request merged**